draft-ietf-tls-tls13.md

```diff
@@ -4609,6 +4653,9 @@ Old versions of TLS permitted the use of very low strength ciphers.
 Ciphers with a strength less than 112 bits MUST NOT be offered or
 negotiated for any version of TLS for any reason.
 
+The security of SSL 3.0 {{SSL3}} is considered insufficient for the reasons enumerated
+in {{RFC7568}}, and MUST NOT be negotiated for any reason.
+
```

Comment:
As I noted in the commit message, the point of the move was to bring sslv2 and sslv3 together while keeping sslv2 and v2-compatible ClientHellos together.  I'm somewhat supportive of combining them, but I think I'll leave it as-is for now and let the editor exercise editorial discretion.